doc/reference/configuration/configuration_reference.rst

```diff
@@ -221,6 +488,718 @@ This section describes options related to :ref:`storing configuration in etcd <c
+    Tarantool restricts how many messages the fibers handle,
+    so that some pending requests are blocked.
+
+    -   On powerful systems, increase ``net_msg_max`` and the scheduler
+        starts processing pending requests immediately.
+
+    -   On weaker systems, decrease ``net_msg_max`` and the overhead
+        may decrease although this may take some time because the
+        scheduler must wait until already-running requests finish.
+
+    When ``net_msg_max`` is reached,
+    Tarantool suspends processing of incoming packages until it
+    has processed earlier messages. This is not a direct restriction of
+    the number of fibers that handle network messages, rather it
+    is a system-wide restriction of channel bandwidth.
+    This in turn restricts the number of incoming
+    network messages that the
+    :ref:`transaction processor thread <thread_model>`
+    handles, and therefore indirectly affects the fibers that handle
+    network messages.
+
+    .. NOTE::
+
+        The number of fibers is smaller than the number of messages because
+        messages can be released as soon as they are delivered, while
+        incoming requests might not be processed until some time after delivery.
+
+    | Type: integer
+    | Default: 768
+    | Environment variable: TT_IPROTO_NET_MSG_MAX
+
+
+.. _configuration_reference_iproto_readahead:
+
+.. confval:: iproto.readahead
+
+    The size of the read-ahead buffer associated with a client connection.
+    The larger the buffer, the more memory an active connection consumes, and the
+    more requests can be read from the operating system buffer in a single
+    system call.
+
+    The recommendation is to make sure that the buffer can contain at least a few dozen requests.
+    Therefore, if a typical tuple in a request is large, e.g. a few kilobytes or even megabytes, the read-ahead buffer size should be increased.
+    If batched request processing is not used, it’s prudent to leave this setting at its default.
+
+    | Type: integer
+    | Default: 16320
+    | Environment variable: TT_IPROTO_READAHEAD
+
+
+.. _configuration_reference_iproto_threads:
+
+.. confval:: iproto.threads
+
+    The number of :ref:`network threads <thread_model>`.
+    There can be unusual workloads where the network thread
+    is 100% loaded and the transaction processor thread is not, so the network
+    thread is a bottleneck.
+    In that case, set ``iproto_threads`` to 2 or more.
+    The operating system kernel determines which connection goes to
+    which thread.
+
+    | Type: integer
+    | Default: 1
+    | Environment variable: TT_IPROTO_THREADS
+
+
+
+
+
+..  _configuration_reference_replication:
+
+replication
+-----------
+
+The ``replication`` section defines configuration parameters related to :ref:`replication <replication>`.
+
+.. NOTE::
+
+    ``replication`` can be defined in the global, group, and replica set :ref:`scope <configuration_scopes>`.
```

Comment:
It is not exactly so.

The only option of this section that is really constrained by a scope is `replication.failover` ([code](https://github.com/tarantool/tarantool/blob/2c422a0b98675cab480ebb44256dfeb72bd2f820/src/box/lua/config/instance_config.lua#L83-L104)).

It is actually have a little sense to set most of the options of the `replication` section for individual instances. However, `replication.anon` is to be used per-instance.

I would move the note about the contraint to `replication.failover` and I wouldn't say anything about `replication` section at whole (because there is no common rule).